core/src/main/java/nl/nn/adapterframework/receivers/Receiver.java

```diff
@@ -1091,20 +1091,20 @@ private void processRawMessage(Object rawMessageOrWrapper, PipeLineSession sessi
 					}
 				}
 				try {
-					technicalCorrelationId = getListener().getIdFromRawMessage((M)rawMessageOrWrapper, session);
+					messageId = getListener().getIdFromRawMessage((M)rawMessageOrWrapper, session);
 				} catch (Exception e) {
 					if(rawMessageOrWrapper instanceof MessageWrapper) { //somehow messages wrapped in MessageWrapper are in the ITransactionalStorage
 						MessageWrapper<M> wrapper = (MessageWrapper)rawMessageOrWrapper;
-						technicalCorrelationId = wrapper.getId();
+						messageId = wrapper.getId(); // TODO: check if this is correct. Isn't the real replyCorrelationId lost in messsageWrapper?
```

Comment:
Waarschijnlijk wel, maar ik neem aan dat wanneer iets in de ErrorStorage komt, het achterliggende systeem het al heeft 'opgegeven' en geen antwoord meer verwacht.